static/app/components/replays/replayController.tsx

```diff
@@ -116,29 +117,52 @@ function ReplayCurrentTime() {
   );
 }
 
-function ReplayPlaybackSpeed({
-  speedOptions,
-  isCompact,
-}: {
-  isCompact: boolean;
-  speedOptions: number[];
-}) {
-  const {setSpeed, speed} = useReplayContext();
+function ReplayOptionsMenu({speedOptions}: {speedOptions: number[]}) {
+  const {setSpeed, speed, isSkippingInactive, toggleSkipInactive} = useReplayContext();
+  const SKIP_OPTION_VALUE = 99999;
```

Comment:
TS should allow a string here, which will never conflict with the speed options.